.github/workflows/podvm_mkosi.yaml

```diff
@@ -192,39 +159,6 @@ jobs:
         if: ${{ inputs.arch == 'amd64' }}
         run: nix build .#devShells.x86_64-linux.podvm-mkosi
 
-      - name: Install crane with nix
-        if: ${{ inputs.arch == 'amd64' }}
-        run: nix profile install nixpkgs#crane
-
-      - name: Install crane with go
```

Comment:
I think we can remove `Setup Golang version ${{ env.GO_VERSION }}` step as it was only needed for building crane? Consequently remove the line were it gets the go version from versions.yaml.

Reply:
Good spot; updated now. Thanks!